specification/eventgrid/resource-manager/Microsoft.EventGrid/preview/2020-04-01-preview/EventGrid.json

```diff
@@ -695,6 +698,260 @@
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/EventChannel"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 400 Bad Request.\n\n * 500 Internal Server Error."
+          }
+        },
+        "x-ms-examples": {
+          "EventChannels_Get": {
+            "$ref": "./examples/EventChannels_Get.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "EventChannels"
+        ],
+        "summary": "Create an event channel.",
+        "description": "Asynchronously creates a new event channel with the specified parameters.",
+        "operationId": "EventChannels_CreateOrUpdate",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "description": "The name of the resource group within the user's subscription.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "partnerNamespaceName",
+            "in": "path",
+            "description": "Name of the partner namespace.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "eventChannelName",
+            "in": "path",
+            "description": "Name of the event channel.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "eventChannelInfo",
+            "in": "body",
+            "description": "EventChannel information.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/EventChannel"
+            }
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/EventChannel"
+            }
+          },
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "#/definitions/EventChannel"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 400 Bad Request.\n\n * 500 Internal Server Error."
+          }
+        },
+        "x-ms-examples": {
+          "EventChannels_CreateOrUpdate": {
+            "$ref": "./examples/EventChannels_CreateOrUpdate.json"
+          }
+        }
+      },
+      "delete": {
```

Comment:
is this a tracked resource ? (ARM routing type)
If so, it must support PATCH (at least tags)

Reply:
No, event channels is a nested resource under partnerNamespaces. its type is:
"Microsoft.EventGrid/partnerNamespaces/eventChannels"